### ISSUE TITLE
Biospecimen BPTL receipt CSV file- Change format of the Date Drawn Column (September Release)

### ISSUE THREAD

**Issue by wyattks** [episphere/connect#983]
The time in the receipt csv for the clinical collections is showing up as 12pm under the Date Drawn column.  We need the format to be 12:00:00 PM.

Shows up like: 01/01/1999 12pm
Needs to be: 01/01/1999 12:00:00 PM

This is for the Clinical collections only.  The research collection time is fine.

**Comment by brotzmanmj** [comment]
@wyattks What's the impact on the BPTL staff effort for this? In order to help us assess the priority?

**Comment by wyattks** [comment]
It's a format change that they need to do in the file.  Nice to have in May but can be pushed to June.

**Comment by wyattks** [comment]
This is fine not to do in June.  Can wait on this.

**Comment by amber-emmes** [comment]
Looking into this. Do you have replication steps for this with data I can use to generate the report? I'm not 100% sure if I'm looking in the right place, and the dates I'm picking aren't finding data, so I can't see from the generated file

**Comment by amber-emmes** [comment]
Just to double-check before I submit this pull request, this is what you would expect that file to look at, and you're specifically asking for a change to the placeholder date?

![image](https://github.com/user-attachments/assets/df724229-37aa-4c38-84a6-7d4cac87cde3)

**Comment by wyattks** [comment]
Yes that is what I would expect to see.  Thank you

**Comment by amber-emmes** [comment]
This has been merged into dev and is ready to test.

**Comment by wyattks** [comment]
Hi @amber-emmes I just tested and I am still gettting the old format in Dev.  I was using an incognito window for the testing.
![Image](https://github.com/user-attachments/assets/295a7e18-8614-4ea5-854b-e6d7c4ee9977)

**Comment by wyattks** [comment]
Never mind- The date Drawn is showing up as expected.  I was looking at the Date Received column!

**Comment by wyattks** [comment]
Tested in Stage.  Ready for Prod.
